### DISCUSSION TITLE
Can you get an example on how to import a javascript package and use it?

### DISCUSSION THREAD

**Discussion by JohnVersus** [reflex-dev/reflex#344] [Q&A]
It is possible to import js packages as we are using bun. Can you provide an example code for importing and using the js package?

**Comment by Alek99** [comment]
Yes see here https://pynecone.io/docs/advanced-guide/wrapping-react

**Reply by JohnVersus** [reply]
Docs only mention about react components. Are the same methods applicable for importing js functions or classes? 

Does pynecode differentiate between components and functions
